overlord/ifacestate/ifacemgr.go

```diff
@@ -36,7 +45,13 @@ func Manager() (*InterfaceManager, error) {
 }
 
 // Connect initiates a change connecting an interface.
-func (m *InterfaceManager) Connect(plugSnap, plugName, slotSnap, slotName string) error {
+//
+func Connect(change *state.Change, plugSnap, plugName, slotSnap, slotName string) error {
+	summary := fmt.Sprintf(i18n.G("Connecting %s:%s to %s:%s"),
```

Comment:
I have mixed feelings about our half-hearted committement to i18n, it seems to mostly buy us .po conflicts. Are we all agreeing to make summary i18n?